client/src/splashPage/FeatureCards.jsx

```diff
@@ -0,0 +1,51 @@
+import React from 'react';
+import styled from 'styled-components';
+import { Card, Tile } from '../styles/globalStyles';
+
```

Comment:
@kalebal Can you go back and make the Cards and the Tiles sections instead of divs? I personally want to maximize the use of semantic HTML where appropriate, what do you think?

Reply:
Yep! I feel like the Tiles are a definitely a good fit for a section tag, but I think Cards may not be since they're more like subsections within Tiles. How would you feel about just changing Tiles for now, and we can come back to Cards once we have the site more fleshed out?